src/coreclr/pal/src/map/virtual.cpp

```diff
@@ -509,6 +510,11 @@ static LPVOID VIRTUALReserveMemory(
              flAllocationType |= MEM_RESERVE_EXECUTABLE;
         }
         pRetVal = ReserveVirtualMemory(pthrCurrent, (LPVOID)StartBoundary, MemSize, flAllocationType);
+
+        if (newMemory != nullptr)
```

Comment:
Can you please change this to `NULL` to be consistent with the rest of the code?